helm-chart/values.yaml.changelog.md

```diff
@@ -5,6 +5,11 @@ For changes that require manual steps other than changing values, please check o
 Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
+## Upgrading to Renku 0.66.0
+
+* NEW `dataService.imageBuilders` to configure session image builds using Shipwright.
+  This feature is experimental and should not be enabled by RenkuLab administrators. Enable the feature at your own risk!
```

Comment:
Same as above - we can at least add one more sentence that enabling this requires Shipwright to be installed independently from the renku helm chart.